### ISSUE TITLE
Needs tests and review for 1.0 release

### ISSUE THREAD

**Issue by sccolbert** [phosphorjs/phosphor-panel#2]

**Comment by sccolbert** [comment]
@afshin @dwillmer please review docs, README, and package dependencies.

**Comment by afshin** [comment]
We have this example in all our repos, but if you `--save-dev` and don't specify `--global` or `-g`, `browserify` will not be in your path. I don't know if that's a big deal, but it might confuse somebody.
```
npm install --save-dev browserify
browserify myapp.js -o mybundle.js
```

**Comment by sccolbert** [comment]
Ah right. It will be on-path if you invoke `browserify` via an npm script. But since we don't depend on browserify, I'm wondering if we should get rid of that section all together?

**Comment by afshin** [comment]
Yeah, I think people using packagers already know, but we should include a note saying something like:

> If you are using a bundler like `browserify` or `webpack`, note that this package `import`s its own CSS file, so your bundler will need to support CSS imports.
